### PR TITLE
Feature/with angular busy

### PR DESCRIPTION
**Pull request by tilt**
https://www.openproject.org/work_packages/5788

### CODE REVIEW

**Comment by myabc** [comment]
@tilt Unfortunately, I'm seeing `No route matches [GET] "/angular-busy/dist/angular-busy.css"` when running Capybara specs locally.

File: app/assets/stylesheets/_external.sass.erb [hunk 1]
```diff
@@ -31,3 +31,4 @@
 
 @import "<%= asset_path("select2/select2.css") %>"
 @import "<%= asset_path("jquery-ui/themes/base/jquery-ui.css") %>"
+@import "<%= asset_path("angular-busy/dist/angular-busy.css") %>"
```
Comment (myabc) [inline app/assets/stylesheets/_external.sass.erb:34]: @tilt the way I did this for select2 and jquery-ui was a hack. We should move to [Bowerify](https://rubygems.org/gems/bowerify) or some custom solution.

File: spec/features/accessibility/work_packages/work_package_query_spec.rb [hunk 2]
```diff
@@ -79,12 +79,12 @@ def column_header_link
       find(column_header_link_selector)
     end
 
-    def sort_ascending_link
-      find(sort_ascending_selector)
```
Comment (myabc) [inline spec/features/accessibility/work_packages/work_package_query_spec.rb:83]: @tilt `find(sort_ascending_selector, visible: true)` would find the element here, but it  Selenium wouldn't allow you to actually click it.

File: app/assets/stylesheets/_external.sass.erb (hunk 1 above)
Comment (manwithtwowatches) [inline app/assets/stylesheets/_external.sass.erb:34]: @tilt any ideas for a fix for that?

Comment (tilt) [inline app/assets/stylesheets/_external.sass.erb:34]: @myabc Does this have to be fixed? And if so any ideas?

Comment (myabc) [inline app/assets/stylesheets/_external.sass.erb:34]: @tilt I think it's beyond the scope of this PR. I'll look into a better solution further down the road. Did you see? https://github.com/opf/openproject/pull/1478#issuecomment-46449730

**Pull request merged**